.github/workflows/build-and-release-stable.yml

```diff
@@ -140,49 +140,48 @@ jobs:
           version: latest
           args: release --clean --skip=validate
       - name: bump homebrew-core formula
-        uses: mislav/bump-homebrew-formula-action@v2
+        uses: mislav/bump-homebrew-formula-action@v3
         with:
           formula-name: mesheryctl
           download-url: https://github.com/meshery/meshery.git
-          formula-path: Formula/m/mesheryctl.rb
```

Comment:
this got fixed per https://github.com/mislav/bump-homebrew-formula-action/pull/59 in 2.3 release